test/unit/token-standard/v2/settlement-factory.test.ts

```
@@ -0,0 +1,615 @@
+      extraArgs: {
+        context,
+        meta: { values: metadataValues },
+      },
+    });
+
+    expect(choiceArgument.extraArgs.context).toEqual(context);
+    expect(choiceArgument.extraArgs.context).not.toBe(context);
+    expect(choiceArgument.extraArgs.context.values).not.toBe(context.values);
+    expect(choiceArgument.extraArgs.meta.values).toEqual(metadataValues);
+    expect(choiceArgument.allocations[0]?.nextIterationFunding).toEqual(funding);
+    expect(Object.prototype.hasOwnProperty.call(choiceArgument.extraArgs.context.values, '__proto__')).toBe(true);
```

ESLint

[error] Use Object.hasOwn() instead of hasOwnProperty

(e18e/prefer-object-has-own)